---
sidebar_position: 5
---

# PEG Evidence Categories

## üåü Why do we need PEG evidence categories?

Predicted Effector Gene (PEG) lists are built from **diverse sources of evidence**, ranging from genetic associations and computational predictions to functional assays and literature curation. Each source has its own origin, methodology, and level of confidence.

To ensure **clarity** and **interoperability**, we group these diverse evidence types into **standardised categories** that are widely recognised in causal gene exploration. This provides a shared framework for describing **variant‚Äìgene**, **variant‚Äìphenotype**, and **gene‚Äìphenotype** relationships within a given phenotype, in a way that is easy to explore and compare.

<span style={{fontSize:"1.3rem", fontWeight:"600", color:"#2E86AB"}}>
‚û°Ô∏è Organising evidence into categories helps users:
</span>
* Interpret consistently ‚Äì understand the general type of support behind each gene.
* Compare across studies ‚Äì align evidence types, even when detailed sources differ.
* Build trust and transparency ‚Äì see at a glance how predictions were made.

---
## PEG Evidence Categories

### üîÄ Variant-centric üîÄ
| Evidence categories (full name) | Abbreviation | Explanation | Real Data example | Real Data example (AMP) ? or example data here? |
|-----------------------------------------|--------------|-------------|-------------------|--------------------------|
| Linkage disequilibrium | LD | Variant is highly correlated with another variant and may act as a proxy. | SNP in LD (r2 > 0.8) with the lead GWAS variant | [Lasconi_sleep_317](https://hugeamp.org/research.html?pageid=Lasconi_sleep_317) |
| Finemapping and credible sets | FM | Finemapping results, or variant is in a credible set with high posterior probability of being causal. | Posterior probability of the index SNP being causal for the GWAS association | [Teumer_UACR_328](https://hugeamp.org/research.html?pageid=Teumer_UACR_328) |
| Colocalisation | COLOC | Variant affects multiple traits (e.g. a complex trait and gene expression data) at the same locus. | Whether the gene is an eGene for a blood eQTL that colocalizes (H4 > 60%) with an asthma-associated SNP | [valette_asthma_287](https://hugeamp.org/research.html?pageid=valette_asthma_287) |
| Molecular QTL | QTL | eQTL, sQTL, pQTL. Variant affects gene expression or splicing or protein level. | Genes for which the lead SNP or its proxies were identified as expression QTLs | [cadby_179](https://hugeamp.org/research.html?pageid=cadby_179) |
| Regulatory region | REG | Variant lies in open chromatin or enhancer/promoter elements in relevant tissue. | "Yes" indicates that the significant SNP is located in a snATAC-seq peak in kidney immune cells | [Liu_2022_eGFR](https://hugeamp.org/research.html?pageid=Liu_2022_eGFR) |
| Chromatin interaction | 3D | Variant physically contacts gene promoter via 3D structure. | Whether the lead SNP or its proxies (r2 ‚â• 0.5) interact with the promoter of the gene, using high-resolution promoter-focused Capture C in human mesenchymal stem cell-derived osteoblasts | [cousminer_bone_accrual_317](https://hugeamp.org/research.html?pageid=cousminer_bone_accrual_317) |
| Predicted functional impact | FUNC | Variant predicted to disrupt gene/protein function or regulatory motifs. | CADD score of the index SNP, from the SNiPA tool (higher CADD scores correspond to greater impact) | [Teumer_UACR_328](https://hugeamp.org/research.html?pageid=Teumer_UACR_328) |
| Proximity to gene (distance) | PROX | Variant is located within or near gene boundaries. | Whether the putative effector gene is the nearest gene to the index SNP | [Verdiesen_AMH_344](https://hugeamp.org/research.html?pageid=Verdiesen_AMH_344) |
| Genome-wide association (GWAS) signal | GWAS | P-value from source GWAS for association of variant with trait specified in metadata file. | p-value for association of the lead SNP with cALT | [vujkovic_NAFLD_305](https://hugeamp.org/research.html?pageid=vujkovic_NAFLD_305) |
| PheWAS (Phenome-Wide Association Study) | PHEWAS | Variant is associated with multiple traits ‚Äî may reveal pleiotropic effects. | UKBB PheWAS Diseases | [cardiogram_165](https://hugeamp.org/research.html?pageid=cardiogram_165) |

### üß¨ Gene-centric üß¨

| Evidence categories | Abbreviation | Explanation | Real Data example | Real Data example (AMP) |
|-----------------------------------------|--------------|-------------|-------------------|--------------------------|
| Protein‚Äìprotein interaction | PPI | Gene‚Äôs protein interacts with other disease-relevant proteins. | Whether the gene product is part of a network of protein-protein interactions predicted to exist among the genes near IgA nephropathy GWAS loci, with p < 0.05 | [Kiryluk_nephropathy_315](https://hugeamp.org/research.html?pageid=Kiryluk_nephropathy_315) |
| Pathway or gene sets | SET | Gene is part of a known pathway or complex relevant to the phenotype. | Whether the gene is predicted by MAGMA pathway analysis (de Leeuw et al. 2015) to participate in a pathway shared among the genes near IgA nephropathy GWAS loci | [Kiryluk_nephropathy_315](https://hugeamp.org/research.html?pageid=Kiryluk_nephropathy_315) |
| Gene-based association | GENEBASE | Gene-based analysis of association of variants (common or rare) in gene with trait. | """The Burden test evidence column contains a classification of the best gene-level p-value from the extreme p-value aggregation test or the weighted aggregation test performed in an exome sequence analysis study of over 49,000 individuals (Flannick, J., et al. 2019). Strong denotes a p-value ‚â§ 5x10-5, while Medium denotes 0.05 > p-value > 5x10-5.""" | [mccarthy_t2d_247](https://hugeamp.org/research.html?pageid=mccarthy_t2d_247) |
| Expression | EXP | Gene is differentially expressed in relevant tissue or patients. e.g. the gene is highly expressed in phenotype-related tissues than others | Level of expression of the gene in the transverse colon, from the GTEx database (transcripts per million, TPM) | [Lasconi_IBD_colonoids_326](https://hugeamp.org/research.html?pageid=Lasconi_IBD_colonoids_326) |
| Perturbation | PERTURB | Gene perturbation causes phenotype-relevant effects in lab or model organisms (knock out animal/cell line, human organoid). | a musculoskeletal (skeleton, limb/digit/tail, and muscle) phenotype included in the MGI mouse phenotype ontology is observed in a knockout of the mouse homolog, as cataloged by Butterfield et al., 2021 or in the MGI Mouse Genome Informatics database. | [OA_effectors_75](https://hugeamp.org/research.html?pageid=OA_effectors_75) |
| Biological Knowledge Inference | KNOW | Gene‚Äìphenotype relationships can be inferred based on known biology, even when there is no direct genetic or experimental evidence linking the specific gene or variant to the new phenotype. No references are provided. |  |  |
| Mendelian Randomization (MR) | MR | Gene‚Äôs predicted expression/protein levels causally affect phenotype. | Genes encoding proteins of established causal relevance to CAD per Mendelian randomization studies, or that are targets for established cardiovascular drugs | [cardiogram_165](https://hugeamp.org/research.html?pageid=cardiogram_165) |
| Genetically predicted trait association (TWAS/PWAS) | TPWAS | Gene‚Äôs genetically predicted expression or protein level is associated with phenotype. Transcriptome- or proteome-wide assocation studies | TWAS (lipid-relevant tissues) | [GLGC_HDL_137](https://hugeamp.org/research.html?pageid=GLGC_HDL_137) |
| Drug related | DRUG | Gene is targeted by drugs known to treat or influence phenotype. | Information from the drug‚Äìgene interaction database (DGIdb) | [kamali_172](https://hugeamp.org/research.html?pageid=kamali_172) |


### Varaint or Gene centric evidence
| Evidence categories| Abbreviation | Explanation | Real Data example | Real Data example (AMP) |
|-----------------------------------------|--------------|-------------|-------------------|--------------------------|
| Cross-phenotype | CROSSP | Gene or variant already established in a related phenotype (biologically similar). | Phenotype with which the index SNP is associated | [Laisk_miscarriage_307](https://hugeamp.org/research.html?pageid=Laisk_miscarriage_307) |
| Literature curation | LIT | Human-curated gene or variant‚Äìdisease links from literature. | Whether there is published evidence implicating the gene in placental function, embryo viability, pregnancy maintenance, or miscarriage/pregnancy loss | [eales_blood_pressure_175](https://hugeamp.org/research.html?pageid=eales_blood_pressure_175) |
| Association from curated database | DB | Variant or Gene is curated as causal or related to the phenotype from existing database, like ClinVar, ClinGen, OMIM, etc. | the gene is linked to a pain or neuropathy disorder in the Online Mendelian Inheritance in Man (OMIM) database. | [OA_effectors_75](https://hugeamp.org/research.html?pageid=OA_effectors_75) |

